internal/kafka/internal/clusters/standalone_provider_test.go

```diff
@@ -1046,3 +1040,202 @@ func TestStandaloneProvider_buildKasFleetshardOperatorSubscription(t *testing.T)
 		})
 	}
 }
+func TestStandaloneProvider_InstallStrimzi(t *testing.T) {
+	type fields struct {
+		connectionFactory      *db.ConnectionFactory
+		dataplaneClusterConfig *config.DataplaneClusterConfig
+	}
+	type args struct {
+		clusterSpec *types.ClusterSpec
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "should InstallStrimzi without error",
+			fields: fields{
+				connectionFactory:      db.NewMockConnectionFactory(nil),
+				dataplaneClusterConfig: mock.BuildValidDataPlaneClusterConfigKasFleetshardOperatorOLMConfig(),
+			},
+			args: args{
+				clusterSpec: &types.ClusterSpec{},
+			},
+		},
+	}
+	RegisterTestingT(t)
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			provider := newStandaloneProvider(test.fields.connectionFactory, test.fields.dataplaneClusterConfig)
+			ok, err := provider.InstallStrimzi(test.args.clusterSpec)
+			Expect(err != nil).To(Equal(test.wantErr))
+			Expect(ok).To(BeTrue())
```

Comment:
if `err != nil` and `test.wantErr == true`, I believe ok won't be `true` and the test will fail.